giacomo_data/jackPapa/agents/2f2f3628$.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```

Comment:
@prefix declaration should end with . 

Change this in every file.

Useful tool to check turtle syntax: http://ttl.summerofcode.be/
```suggestion
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
```